src/main/java/org/myrobotlab/programab/Session.java

```diff
@@ -82,6 +83,11 @@ private synchronized Chat getChat() {
         predicatesFile = userPredicates;
         chat.predicates.getPredicateDefaults(userPredicates.getAbsolutePath());
       }
+      
+      ProgramABConfig config = (ProgramABConfig)programab.getConfig();
+      if (config.startTopic != null){

```

Comment:
I don't think this is the right thing to do here.  When a session is first created with a new user..  someone that hasn't been seen before.. it makes sense to set the start topic..  (which should be unknown by default) ...  
Think of the scenario where the robot is talking with multiple people...   you don't want to reset the topic each time the user switches between sessions.

Reply:
I'm not sure I agree.  Yes multi session is a complexity to begin addressing in a meaningful way, but right now there is just "unknown" as far as the consistency of starting a topic.  I think there is value in being able to consistently start a new session with a known topic that isn't "unknown" depending the the user's needs